src/julia/devcontainer-feature.json

```diff
@@ -18,6 +18,9 @@
     "installsAfter": [
         "ghcr.io/devcontainers/features/common-utils"
     ],
+    "postCreateCommand": {
+        "julia-activate": "/usr/local/julia-devcontainer-features/postcreate.jl"
+    },
```

Comment:
Thinking again, would this be appropriate with `postCreateCommand`, which is executed only once when the container is created.
`postStartCommand` or `postAttachCommand` would be executed each time the container is started
https://containers.dev/implementors/json_reference/

Reply:
I think running the script when the container is created should be enough? Testing with a package, once ``Pkg.develop()`` is called on container creation, that seems to persist across sessions.

Reply:
Ok, thanks for checking this!